### DISCUSSION TITLE
👋 Welcome to discord-economy-super Discussions!

### DISCUSSION THREAD

**Discussion by shadowplay1** [shadowplay1/discord-economy-super#24] [Announcements]
## 👋 Welcome!
This repository now has a Discussions! 💪
I will upload all the important announcemnts and releases here 📢
Feel free to create your own discussion about anything related to this module! Share your ideas, bugs, feedback or anything! 💡

**Comment by MegaSa1nt** [comment]
WTF shadowplay1 NVIDIA?!?!!
